pandas/core/groupby/groupby.py

```diff
@@ -171,6 +171,10 @@ class providing the base-class of operations.
         The resulting dtype will reflect the return value of the passed ``func``,
         see the examples below.
 
+    Functions that mutate the passed object can produce unexpected
```

Comment:
this should be in Notes (wouldn't object to having this as a warning)

Reply:
My apologies, I'm new to contributing. Did I put my additional note in the incorrect 'Notes' section?

Reply:
Looks correct to me.